accounts_test.go

```diff
@@ -670,7 +672,10 @@ func TestRemoveAccountKey(t *testing.T) {
 	result, err = b.ExecuteNextTransaction()
 	assert.NoError(t, err)
 
-	unittest.AssertFVMErrorType(t, &fvmerrors.InvalidProposalSignatureError{}, result.Error)
+	unittest.AssertFVMErrorType(t,
+		&fvmerrors.InvalidProposalSignatureError{},
+		result.Error.(*types.SignatureError).Unwrap(),
+	)
```

Comment:
Instead of force-casting, use e.g. `require.ErrorAs` in a separate line before. Same in the the assertions below

Reply:
Good idea, thank you